prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -954,5 +954,51 @@ public async Task<IEnumerable<string>> GetEnrolleeEmails(BulkEmailType bulkEmail
                 .DecompileAsync()
                 .ToListAsync();
         }
+
+        public async Task<bool> IsPotentialPaperEnrolleeReturnee(string dateOfBirth)
+        {
+            return await _context.Enrollees
+                .AsNoTracking()
+                .AnyAsync(e => e.GPID.StartsWith("NOBCSC")
```

Comment:
We should avoid using magic strings ("NOBCSC") whenever possible. It is defined in the `EnrolleePaperSubmissionService`; which is part of the reason I'm thinking these methods should be in that service instead.

Reply:
Moved and used the GPID filter in `EnrolleePaperSubmissionService`